app/visualization.py

```diff
@@ -0,0 +1,67 @@
+from app.requester import ExchangeRateRequester
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+class Grapher:
+    def __init__(self) -> None:
+        self.exchange_rate_requester = ExchangeRateRequester()
+
+    def _create_historical_rates_df(self, transactions) -> pd.DataFrame:
+        assert transactions, "There are no transactions"
+
+        first_transaction = transactions[0].date
+        historical_rates_date = self.exchange_rate_requester.get_historical_bids(first_transaction)
+        historical_rates_df = pd.DataFrame(list(historical_rates_date.items()), columns=["date", "rate"])
+
+        return historical_rates_df
+
+    def _create_operations_df(self, transactions: list) -> pd.DataFrame:
```

Comment:
you can extract some methods from this function ;)
It's quite long.

Reply:
done